iceoryx_binding_c/test/moduletests/test_subscriber.cpp

```diff
@@ -82,10 +82,16 @@ class iox_sub_test : public Test
     iox::mepoo::SharedChunk getChunkFromMemoryManager()
     {
         constexpr uint32_t PAYLOAD_SIZE{100U};
-        return m_memoryManager.getChunk(PAYLOAD_SIZE,
-                                        iox::CHUNK_DEFAULT_PAYLOAD_ALIGNMENT,
-                                        iox::CHUNK_NO_CUSTOM_HEADER_SIZE,
-                                        iox::CHUNK_NO_CUSTOM_HEADER_ALIGNMENT);
+
+        auto chunkSettingsResult = ChunkSettings::create(PAYLOAD_SIZE, iox::CHUNK_DEFAULT_PAYLOAD_ALIGNMENT);
+        EXPECT_FALSE(chunkSettingsResult.has_error());
```

Comment:
`ASSERT_FALSE`?

Reply:
not possible, see https://github.com/eclipse-iceoryx/iceoryx/pull/629#discussion_r599545927